binding/python/raftify.pyi

```diff
@@ -79,7 +108,11 @@ class Slogger:
     def default() -> "Slogger": ...
     @staticmethod
     def new_file_logger(
-        log_path: str, chan_size: int, rotate_size: int, rotate_keep: int
+        log_path: str,
+        level: "Level" = Level.DEBUG,
+        chan_size: int = 3000,
```

Comment:
If it's difficult to provide a sensible default value, let's have the user input a value instead of providing a default. (`chan_size`)